dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -18,9 +18,30 @@ resource "kafka_topic" "uswitch_data_v1" {
   }
 }
 
+resource "kafka_topic" "uswitch_events_v1" {
+  name = "customer-proposition.uswitch.events.v1"
+
+  replication_factor = 3
+  partitions         = 15
+
+  # infinte retention
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
```

Comment:
Do you need infinite retention in dev?

Reply:
Maybe not "need" but I think there's value in having infinite retention in dev - the way the system is architected is that kafka is the long-term storage for our data. I also think there's value in dev config matching prod config. I can ask my team and see if they have a different opinion though. 

Our current topic in dev is configured for infinite retention, I'm assuming we wouldn't want to change it as part of the migration? I.e. we'd want to change that before we migrate all the data or after we migrate all the data?

Reply:
No worries, we can stick with infinite retention, just raised it for it to be a conscious decision